src/ResourceManager/RecoveryServices.SiteRecovery/Commands.RecoveryServices.SiteRecovery/AlertSetting/SetAzureRmRecoveryServicesAsrAlertSetting.cs

```diff
@@ -36,16 +36,19 @@ namespace Microsoft.Azure.Commands.RecoveryServices.SiteRecovery
     public class SetAzureRmRecoveryServicesAsrAlertSetting : SiteRecoveryCmdletBase
     {
         /// <summary>
-        ///     Gets or sets EmailSubscriptionOwner .Mail to subscription owner.
+        ///     Switch paramter specifies enable notification to subscription owner.
```

Comment:
This I will take separately in other CR. Using help message to define Md help.
If I will do it now it will touch each parameter and increase the size of this pull request.

Reply:
https://github.com/Azure/azure-powershell/issues/5640